rusk-prover/src/lib.rs

```diff
@@ -10,21 +10,119 @@ extern crate alloc;
 #[cfg(feature = "std")]
 extern crate std;
 
+use alloc::format;
 use alloc::vec::Vec;
 
-mod errors;
-mod tx;
+use dusk_bytes::{Error as BytesError, Serializable};
+use dusk_plonk::prelude::Prover as PlonkProver;
+use once_cell::sync::Lazy;
 
-#[cfg(feature = "local_prover")]
-pub mod prover;
-#[cfg(feature = "local_prover")]
-pub use crate::prover::LocalProver;
```

Comment:
This was under feature flag in order to let custom prover implementation to use prover lib without the need of local prover (check our lambda prover in `dusk_prover` repository)

Reply:
Since the `rusk-prover::Prover` trait is superseded by `execution-core`s `Prove` trait, the library now only contains the `LocalProver` (which implements the `Prove` trait). That way the feature structure in rusk-prover is massively simplified.